src/xdocs/config_design.xml

```diff
@@ -1653,11 +1894,11 @@ public class InputPublicImmutable {
         </p>
         <source>
 public class InputPublicImmutable {
-  public final int someIntValue;
-  public final ImmutableSet&lt;String&gt; includes;
-  public final java.lang.String notes;
-  public final BigDecimal value;
-  public final List list;
+  public final int someIntValue; // Ok
+  public final ImmutableSet&lt;String&gt; includes; // Ok
+  public final java.lang.String notes; // Ok
+  public final BigDecimal value; // Ok
+  public final List list; // Ok
```

Comment:
Let's be consistent with `// violation`, please change it to `// ok`

Reply:
done

Reply:
Resolved.